core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -165,12 +124,141 @@ object ConsumerGroupCommand extends Logging {
                                                 consumerId: Option[String], host: Option[String],
                                                 clientId: Option[String], logEndOffset: Option[Long])
 
+  protected case class MemberAssignmentState(group: String, consumerId: Option[String], host: Option[String],
+                                             clientId: Option[String], numPartitions: Option[Int],
+                                             assignment: Option[List[TopicPartition]])
+
+  protected case class GroupState(group: String, assignmentStrategy: Option[String], state: Option[String], numMembers: Option[Int])
+
   sealed trait ConsumerGroupService {
 
     def listGroups(): List[String]
 
-    def describeGroup(): (Option[String], Option[Seq[PartitionAssignmentState]]) = {
-      collectGroupAssignment(opts.options.valueOf(opts.groupOpt))
+    private def checkGroup(group: String, state: Option[String], numRows: Option[Int]): Boolean = {
```

Comment:
Could we choose a more descriptive name? Maybe `isGroupActive` or something like that? Also, not really obvious what purpose `numRows` is serving since it's use is a bit indirect.